projects/storefrontlib/src/cms-structure/routing/default-routing-config.ts

```diff
@@ -219,6 +219,9 @@ export const defaultStorefrontRoutesConfig: RoutesConfig = {
   userGroupAssignUsers: {
     paths: ['organization/user-group/assign-users/:code'],
   },
+  orderApprovals: {
+    paths: ['my-account/approval-dashboard'],
```

Comment:
isn't this for the listing page?

Reply:
Exact, I added this for the "Back to List" to work.

Reply:
got it